plugin.config.json

```diff
@@ -2,23 +2,24 @@
     "headers": {
         "prod": {
             "downloadURL": "https://cdn2.wasabee.rocks/iitcplugin/prod/wasabee.user.js",
-            "updateURL": "https://cdn2.wasabee.rocks/iitcplugin/prod/wasabee.meta.js"
+            "updateURL": "https://cdn2.wasabee.rocks/iitcplugin/prod/wasabee.meta.js",
+            "version": "0.20.BUILDDATE"
         },
         "dev": {
             "downloadURL": "https://am.wasabee.rocks/static/dev/wasabee.user.js",
-            "updateURL": "https://am.wasabee.rocks/static/dev/wasabee.meta.js"
+            "updateURL": "https://am.wasabee.rocks/static/dev/wasabee.meta.js",
+            "version": "0.20.BUILDDATE-dev"
```

Comment:
Not for this, but when we get some time to research it. Could we add the username of the builder and/or the branch name?

Reply:
branch name sounds feasible